docs/ConvertRBAC.md

```diff
@@ -0,0 +1,45 @@
+# Convert RBAC policies
+
+There's an RBAC converter that works on ClusterRoleBindings or RoleBindings.
```

Comment:
I'll follow up with a PR to add some explanation (like how plain RBAC is going to be faster by nature of not needing an external webhook call, but that this is to facilitate migration of policies to be more restrictive)

Reply:
Cool, let me know if you need review on that before merging.